src/io/fasta.rs

```diff
@@ -201,24 +202,21 @@ where
     /// # Example
     ///
     /// ```rust
-    /// # use std::error::Error;
-    /// # use bio::io::fasta::{Reader, FastaRead};
-    /// # use bio::io::fasta::Record;
-    /// # fn main() -> Result<(), Box<Error>> {
+    /// use bio::io::fasta::{Reader, FastaRead};
+    /// use bio::io::fasta::Record;
+    ///
     /// const fasta_file: &'static [u8] = b">id desc
     /// AAAA
     /// ";
     /// let mut reader = Reader::new(fasta_file);
     /// let mut record = Record::new();
     ///
     /// // Check for errors parsing the record
-    /// reader.read(&mut record)?;
+    /// reader.read(&mut record).expect("Error: broken record detected");
```

Comment:
Maybe make this even more specific:
```suggestion
    /// reader.read(&mut record).expect("fasta reader: got an io::Error or could not read_line()");
```